server/scripts/import-tests/index.js

```diff
@@ -37,27 +37,24 @@ const tmpDirectory = path.resolve(__dirname, 'tmp');
 const testDirectory = path.resolve(tmpDirectory, 'tests');
 const supportFile = path.resolve(testDirectory, 'support.json');
 
-const ariaat = {
+const ariaAtImport = {
```

Comment:
Thank you for changing that name, haha, this is much better.

Reply:
Small (perhaps nitpicky) thing: we use "main" as our primary branch, not "master."

Reply:
Right, will adjust

Reply:
Actually, not seeing the `main` branch for aria-at. And `master` is still set as default so I suppose that branch rename could be proposed there?

I'll change the verbiage here to reflect a 'default branch' and set a constant for it should that change occur